docs/pages/database-access/guides/aws-dynamodb.mdx

```diff
@@ -0,0 +1,275 @@
+
+![Create Role Step 1](../../../img/database-access/guides/dynamodb-create-role-1.png)
+
+Click **Next**. Find the AWS-managed policy `AmazonDynamoDBFullAccess` and then select the policy:
+
+![Create Role Step 2](../../../img/database-access/guides/dynamodb-create-role-2.png)
+
+<Admonition type="note" title="Apply least-privilege permissions">
+The `AmazonDynamoDBFullAccess` policy may grant more permissions than desired.
+If you want to use a different IAM policy to reduce permissions, refer to
+[Managing access permissions to your Amazon DynamoDB 
+Resources](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/access-control-overview.html)
+for more information.
+</Admonition>
+
+Click **Next**. On the next page, enter a role name. In this guide we'll use
+the example name `ExampleTeleportDynamoDBRole` for this role.
+
+Under "Select trusted entities", update the JSON to allow the `TeleportDatabaseService`
+role to assume this role:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": [
+                    "arn:aws:iam::(=aws.aws_access_key=):role/TeleportDatabaseService"
+                ]
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {}
+        }
+    ]
+}
+```
+
+Finally, click **Create Role**.
+
+## Step 2/4. Configure the Teleport IAM role mapping
+
+The next step is to give your Teleport users permissions to assume IAM roles in
+your Teleport cluster.
+
+You can do this by creating a Teleport role with the `db_users` field
+listing the IAM role ARN created in the previous step. Create a file called
+`aws-dynamodb-access.yaml` with the following content:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: aws-dynamodb-access
+spec:
+  allow:
+    db_labels:
+      '*': '*'
+    db_users:
+    - 'ExampleTeleportDynamoDBRole' # only the role name, do not specify the full AWS ARN
+```
+
+Create the new role:
+
+```code
+$ tctl create -f aws-dynamodb-access.yaml
+```
+
+(!docs/pages/includes/add-role-to-user.mdx role="aws-dynamodb-access"!)
+
+## Step 3/4. Install the Teleport Database Service
+
+Create an EC2 instance to host the Teleport Database Service, and attach the
+`TeleportDatabaseService` AWS IAM role to it. If you're hosting the service another
+way, you must provide AWS credentials to the service - see [AWS credentials
+configuration](https://docs.aws.amazon.com/sdkref/latest/guide/creds-config-files.html)
+for more details.
+
+<Admonition type="warning" title="non-standard AWS regions">
+For non-standard AWS regions such as AWS GovCloud (US) regions and AWS China
+regions, please set the corresponding region in the `AWS_REGION` environment
+variable or in the AWS credentials file so that the Database Service can use
+the correct STS endpoint.
+</Admonition>
+
+### Generate a token
+
+A join token is required to authorize a Teleport Database Service instance
```

Comment:
Since this guide uses an EC2 instance, it's pretty safe to assume the reader is on AWS. IAM joining and EC2 joining are both more secure than static tokens.

I'm fine if we want to keep the static token here since it's less to explain that setting up IAM joining (which has its own guide), but maybe we should just mention there are other more secure methods and soften the "required" wording a bit?

Reply:
+1 for mentioning the more secure methods, but keeping the static token method for simplicity

Reply:
Please confirm, are you referring to these two methods?
- https://goteleport.com/docs/server-access/guides/ec2-discovery/
- https://goteleport.com/docs/management/guides/joining-nodes-aws-iam/

Reply:
Yep

Reply:
See 0fdbfce9c6a73ad77d11b7701b36d4c7ffc73f7d